src/ft/mod.rs

```diff
@@ -427,7 +427,8 @@ impl FreeTypeRasterizer {
         // Render a normal character if it's not a cursor.
         let font_key = self.face_for_glyph(glyph_key)?;
         let face = &self.faces[&font_key];
-        let index = face.ft_face.get_char_index(glyph_key.c as usize);
+        let index = face.ft_face.get_char_index(glyph_key.character as usize);
+        let is_missing_glyph = index == 0;
```

Comment:
Index is never reassigned, right? So we should be able to just inline this check?

Also at this point I'm fairly certain we want a constant for this (for each platform, I think a global one is a bad idea even tough it seems to be the same on each platform).